NetTopologySuite.IO.Oracle.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+	<metadata>
+		<id>NetTopologySuite.IO.Oracle</id>
+		<version>1.15.1.0</version>
+		<title>NetTopologySuite IO Library for Oracle</title>
+		<authors>Holger Laebe</authors>    
+		<license type="expression">LGPL-2.1-or-later</license>
+		<projectUrl>https://github.com/hlaebe/NetTopologySuite.IO.Oracle</projectUrl>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<description>
+      This package contains the Oracle IO library with experimental writer implementation
+		</description>
+		<summary>Oracle IO library for NetTopologySuite</summary>
+		<tags>NTS Oracle</tags>
+		<language>en-US</language>
+		<frameworkAssemblies>
+			<frameworkAssembly assemblyName="System" targetFramework="" />
+			<frameworkAssembly assemblyName="System.Core" targetFramework="" />
+		</frameworkAssemblies>
+		<dependencies>
+			<dependency id="NetTopologySuite" version="[1.15.1.0]" />
```

Comment:
This should be `version="1.15.1"`, or perhaps `version="[1.15.1,2.0)"`

Reply:
Agreed